fake-server/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express")
+const bodyParser = require('body-parser');
+const fakeUsers = require("./fake-users.json");
+const app = express();
+const port = 3000
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.post("/", (req, res) => {
+    const { username, password } = req.body || {};
+    var policy = "";
+    if(!username || !password) {
+        res.status(400).send("Bad request");
+    } else {
+        let user = fakeUsers[username];
+        if(!user) {
+            res.status(403).send("Login failed");
+            console.log("Login failed for unknown user", username);
+        } else {
+            if(user.password = password) {

```

Comment:
if (user.password === password) {

Reply:
ok thanks! Happy to fix any other errors you spot. @steviej08 will also review to find what he can.

Reply:
fixed